{
  try {
    await().atMost(5,TimeUnit.SECONDS).until(new Callable<Boolean>(){
      @Override public Boolean call() throws Exception {
        return callback.isSuccess();
      }
    }
);
  }
 catch (  Exception e) {
    Assert.fail(error,e);
  }
}
